components/slack/actions/reply-to-a-message-thread/reply-to-a-message-thread.mjs

```diff
@@ -1,13 +1,15 @@
 import slack from "../../slack.app.mjs";
+import common from "../send-message-common.mjs";
 
 export default {
+  ...common,
   key: "slack-reply-to-a-message",
   name: "Reply to a Message Thread",
   description: "Send a message as a threaded reply",
```

Comment:
All actions need to have a link to the relevant documentation in the description. This comment applies to all action components in this and all apps.